ldk/node/src/hostClients/whisperMessageBuilder.ts

```diff
@@ -236,11 +238,56 @@ export const generateWhisperListElement = (
   return msg;
 };
 
+export const generateWhisperDisambiguationElement = (
+  element: WhisperDisambiguationElements,
+): messages.WhisperDisambiguationElement => {
+  const WDE = messages.WhisperDisambiguationElement;
```

Comment:
Instead of assigning as a constant here, perhaps just create it with the namespace?
```suggestion
  const msg = new messages.WhisperDisambiguationElement();
```

Reply:
Though if you're doing this to save typing out the whole class each time, maybe assign it outside of the function?

IDK, I have a preference against assigning the same variable repeatedly inside a function if it's not changing inside the function, but that's just my preference.

Reply:
This is how the `generateWhisperListElement()` function is set up, but I agree with you. I didn't want to change anything just in case I caused some sort of unforeseen issue with how this works.